### DISCUSSION TITLE
Make the removal of colors in the build output optional

### DISCUSSION THREAD

**Discussion by LoiLock** [vercel/next.js#61936] [Ideas]
### Goals

1. The changes made in #60385 are opinionated. Personally, I care a lot for the bundle-size of our project, and very often the colors in the build output have helped me find accidental imports, and reduce the bundle-size

### Background

I would like for this to be a config option

### Proposal

Not too sure, allow to set the thresholds for the green, orange, and red text would be good

**Comment by FadiObaji** [comment]
Seriously why removing it? At least it should be an option.
